app/models/garden.rb

```diff
@@ -1,3 +1,10 @@
 class Garden < ApplicationRecord
    has_many :plots
+
+   def self.one_hundred_or_less_harvest_days
```

Comment:
This might be easier if you make it an instance method! We're getting plants for a specific garden